src/testUtils.ts

```diff
@@ -89,10 +90,64 @@ export function getTestFunctions(doc: vscode.TextDocument, token: vscode.Cancell
 		.then(symbols =>
 			symbols.filter(sym =>
 				sym.kind === vscode.SymbolKind.Function
-				&& (sym.name.startsWith('Test') || sym.name.startsWith('Example')))
+				&& (sym.name.startsWith('Test') || sym.name.startsWith('Example') || testSuiteMethodRegex.test(sym.name))
+			)
 		);
 }
 
+/**
+ * Extracts test method name of a suite test function.
+ * For example a symbol with name "(*testSuite).TestMethod" will return "TestMethod".
+ *
+ * @param symbolName Symbol Name to extract method name from.
+ */
+export function extractInstanceTestName(symbolName: string): string {
+	const match = symbolName.match(testSuiteMethodRegex);
+	if (!match || match.length !== 3) {
+		return null;
+	}
+	return match[2];
+}
+
+/**
+ * Finds test method containing suite.Run() call for the given instance test name.
+ *
+ * @param symbolName Instance test function name (including the type name)
+ * @param doc Editor document
+ * @param allTests All test functions
+ */
+export function findTestFnForInstanceTest(symbolName: string, doc: vscode.TextDocument, allTests: vscode.SymbolInformation[]): vscode.SymbolInformation {
+	// get non-instance test functions
+	const testFunctions = allTests.filter(t => !testSuiteMethodRegex.test(t.name));
+	// filter further to ones containing suite.Run()
+	const candidates = testFunctions.filter(t => {
+		const text = doc.getText(t.location.range);
+		return /suite\.Run\(/.test(text);
```

Comment:
Since this is a simple string check, wont `text.includes('suite.Run(')` be simpler?
Unless we make this a useful regex like `/suite\.Run\([^\)]+\)/